dev/NavigationView/NavigationView.cpp

```diff
@@ -1707,9 +1702,10 @@ winrt::IInspectable NavigationView::MenuItemFromContainer(winrt::DependencyObjec
 {
     if (const auto& nvi = container)
     {
+        winrt::IInspectable item{ nullptr };
```

Comment:
We usually try to avoid initializing variable to dummy values, in this case I think it can be avoided. This is older code was doing this before your change but maybe while you are here you can change it?

```
winrt::IInspectable NavigationView::MenuItemFromContainer(winrt::DependencyObject const& container)
{
    if (const auto& nvi = container)
    {
        if (IsTopNavigationView())
        {
            // Search topnav first, if not found, search overflow
            if (const auto lv = m_topNavListView.get())
            {
                if (const auto item =  lv.ItemFromContainer(nvi))
                {
                    return item;
                }
            }

            if (const auto lv = m_topNavListOverflowView.get())
            {
                if (const auto item = lv.ItemFromContainer(nvi))
                {
                    return item;
                }
            }

            // if still not found search footer
            if (const auto flv = m_topFooterNavListView.get())
            {
                return flv.ItemFromContainer(nvi);
            }

            return nullptr;
        }
        else
        {
            if (const auto lv = m_leftNavListView.get())
            {
                if (const auto item lv.ItemFromContainer(nvi))
                {
                    return item;
                }

                // if not found search footer
                if (const auto flv = m_leftFooterNavListView.get())
                {
                   return flv.ItemFromContainer(nvi);
                }
            }
            return nullptr;
        }
    }

    return nullptr;
}
```